Sources/eppo/Constants.swift

```diff
@@ -1,5 +1,5 @@
 // todo: make this a build argument (FF-1944)
 public let sdkName = "ios"
-public let sdkVersion = "4.2.0"
+public let sdkVersion = "4.3.0"
```

Comment:
We often bump the version separately from PRs that introduce changes, but if Swift packages are loaded directly from GH, then I probably should have bumped this in https://github.com/Eppo-exp/eppo-ios-sdk/pull/61